src/components/StyleGuide/ColorMode.stories.mdx

```diff
@@ -0,0 +1,201 @@
+
+The `useColorMode` hook is provided by Chakra to allow managing the color mode
+in an application. This hook is also exported by the DS. Invoking `useColorMode`
+will return an object with the following properties:
+
+- `colorMode` is the current color mode of the application as "light" or "dark".
+- `toggleColorMode` is a function that can be used to toggle the color mode.
+
+```tsx
+import { Button, useColorMode } from "@nypl/design-system-react-components";
+
+const ToggleColorModeButton = () => {
+  const { colorMode, toggleColorMode } = useColorMode();
+
+  return (
+    <>
+      <p>The current color mode is {colorMode}</p>
+      <Button id="toggleButton" onClick={toggleColorMode}>
+        Toggle Color Mode
+      </Button>
+    </>
+  );
+};
+```
+
+### useColorModeValue
+
+The `useColorModeValue` hook is provided by Chakra to allow setting style values
+based on the app's color mode. This hook is also exported by the DS. The two
+arguments passed to `useColorModeValue` are the light mode value and the dark
+mode value. This hook does _not_ update the app's color mode; it returns the
+appropriate value based on the current color mode.
+
+As an example, let's make the `Heading` component color mode aware:
+
+```tsx
+import { Heading, useColorModeValue } from "@nypl/design-system-react-components";
+
+const AppPage = () => {
+  const lightModeValue = "ui.black";
+  const darkModeValue = "ui.white";
+  const headingColor = useColorModeValue(lightModeValue, darkModeValue);
+
+  return (
+    <Heading level={1} color={headingColor}>
+  );
+};
+```
+
+## Using Local Storage to Manage Color Mode
+
+### ColorModeScript
+
+This component is used to sync local storage correctly for the apps's color mode.
+This should be added in the app's `body` tag before all the content.
+
+#### Basic App Example
+
+This example is based on using Parcel for bundling and running the app but it
+will work for any basic front-end app architecture. Code shorten for brevity.
+
+```tsx
+// src/index.tsx
+import { ColorModeScript } from "@nypl/design-system-react-components";
+import { App } from "./components/App";
+
+const app = document.getElementById("app");
+ReactDOM.render(
+  <>
+    <ColorModeScript />
+    <App />
+  </>,
+  app
+);
+```
+
+#### Next.js App Example
+
+The `ColorModeScript` component should be added in the `_document.tsx` file.
+
+```tsx
+// pages/_document.js
+import { ColorModeScript } from "@nypl/design-system-react-components";
+import { Html, Head, Main, NextScript } from "next/document";
+
+const Document = () => {
+  const initialColorMode = "light";
+  render() {
+    return (
+      <Html lang="en">
+        <Head />
+        <body>
+          <ColorModeScript initialColorMode={initialColorMode} />
+          <Main />
+          <NextScript />
+        </body>
+      </Html>
+    );
+  }
+};
+
+export default Document;
+```
+
+### colorModeManager
+
+This is a prop that is passed to the `DSProvider` component. It is most helpful
```

Comment:
After reading this, I am not sure what this is used for.  Is there any more info you can add here for clarity?  Or will devs with more React experience understand what this is all about?